src/_gettsim/parameters/eink_st_abzuege.yaml

```diff
@@ -408,11 +408,13 @@ einführungsfaktor:
     en: Introductory factor for pension expense
   description:
     de: >-
-      In den Jahren 2005-2025 steigt der Anteil der abzugsfähigen Beiträge von 60% in
```

Comment:
I think it should be clear from the docstring that this was a discrete change to a pre-announced path (see [comment](https://github.com/iza-institute-of-labor-economics/gettsim/issues/491#issuecomment-1400276767) in #491) -- whether we actually do anything with that, we shall see.

Reply:
good point. Done.